lib/ProductOpener/Display.pm

```diff
@@ -1614,12 +1614,12 @@ sub set_cache_results ($key, $results) {
 
 	if ($memd->set($key, $results, 3600)) {
 		$mongodb_log->info("set_cache_results - updated - key: $key - uncompressed total_size: $result_size")
```

Comment:
Ah I made a mistake, I wanted to keep those so that we can have cache stats easily in prod:

```
off@off:/srv/off/logs$ tail -n 100000 mongodb_log4perl.log | ./cache_stats.pl 
Stats for PRODUCTS_COUNT queries:
- get: 24615
- get - hit: 2548 (10.4%)
- get - miss: 21907 (89.0%)
- set: 19356
- set - updated: 19356 (100.0%)
- set - skipping: 0 (0.0%)
Stats for PRODUCTS queries:
- get: 30365
- get - hit: 4844 (16.0%)
- get - miss: 25249 (83.2%)
- set: 21938
- set - updated: 20262 (92.4%)
- set - skipping: 1676 (7.6%)
```